### ISSUE TITLE
NRIC Generator Implementation on Website uses an incorrect algorithm

### ISSUE THREAD

**Issue by bigbadmonster17** [samliew/singapore-nric#4]
Topic: 
Hi! Just wanted to let you know that the 'M' series is only for Foreigners. This means that generating NRICs for Singaporean Citizens and PRs born from 2020-future will still use the 'T' series. I write to you because selecting 'T' + '2020-2030' is not possible and will result to changing 'T' to 'M' automatically. I've looked at both nric.js and the respective test cases... it may be my opinion but there seems to be nothing wrong with your repository so i can only assume it could be the implementation on your website not allowing me to have this specific selection. Please find below a link to the ICA press release on the M FIN series being for foreigners. Hope you have a great day!

https://www.ica.gov.sg/news-and-publications/newsroom/media-release/new-m-fin-series-to-be-introduced-from-1-january-2022

**Comment by sheen4n** [comment]
@bigbadmonster17 that sounds about right.

TLDR: 
- S and T series are for Singapore Citizens and PRs
- F,G and M are for foreigners FIN?

@samliew , how do go about contributing to this package to provide a fix? :)

**Comment by samliew** [comment]
Website has to be fixed manually atm, I'll do it when I have time. Thanks for letting me know
